fdbclient/SystemData.cpp

```diff
@@ -343,13 +340,28 @@ const Key storageServerListToTeamIdKey(std::vector<UID> servers) {
 }
 
 const Value encodeStorageTeams(const std::vector<UID>& value) {
+	return BinaryWriter::toValue(value, IncludeVersion());
+}
+
+std::vector<UID> decodeStorageTeams(const ValueRef& value) {
+	return BinaryReader::fromStringRef<std::vector<UID>>(value, IncludeVersion());
+}
+
+const KeyRef storageTeamIdToTLogGroupPrefix = LiteralStringRef("\xff/storageTeamIdToTLogGroup/");
```

Comment:
FYI. A shorthand for `LiteralStringRef("\xff/storageTeamIdToTLogGroup/")` is `"\xff/storageTeamIdToTLogGroup/"_sr`.